std_msgs/Header header
int64 class_id
float64 score
int64 track_id
sensor_msgs/Image mask
